### PR TITLE
Some cleanup on Home.js

### PR DESCRIPTION
**Pull request by amielmon**
-Arrowify event listener functions and remove binds
-URL properly changes when clicking on logo
-Replace open/close for feedback and help forms with single toggle

### CODE REVIEW

**Comment by gamburgm** [comment]
Looking good to me! Going to get @dajinchu to take a closer look since I'm less experienced with the stuff, but glad we get to get this code in.

**Comment by dajinchu** [comment]
Looks good. Could you update the snapshots? Tests are failing.

**Comment by ryanhugh** [comment]
> -Arrowify event listener functions and remove binds

People have asked me about this before and I originally didn't support moving over because sublime text didn't use to recognize functions with this new syntax as methods (so shortcuts like ctrl-r didn't work) but looks like sublime now supports them! Woo!!

You could probably make these changes throughout the rest of the codebase too, if ya wanted :)

Great work!!

File: backend/graphql/tests/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -3,19 +3,11 @@
 exports[`gets all occurrences 1`] = `
```
Comment (dajinchu) [inline backend/graphql/tests/__snapshots__/index.test.js.snap]: can you undo the changes to this snapshot

**Review by dajinchu — CHANGES_REQUESTED** [review]

**Review by dajinchu — APPROVED** [review]
👍

**Pull request merged**